particles/List/source/Items.js

```diff
@@ -66,25 +61,33 @@ defineParticle(({DomParticle, resolver, html}) => {
     shouldRender({list}) {
       return Boolean(list);
     }
+    getId(entity) {
+      // TODO(sjmiles): ensure we get the entity.id even if the rawData has it's own id,
+      // discuss with Runtime folks (either we disallow `id` collision or have a more obvious way to get this)
```

Comment:
Maybe create a GH issue for this and the usage below so it's not forgotten?

Reply:
#3147